lms/djangoapps/ora_staff_grader/views.py

```diff
@@ -147,6 +150,49 @@ def get(self, request, ora_location, *args, **kwargs):
             return UnknownErrorResponse()
 
 
+class FeedbackListView(StaffGraderBaseView):
+    """
+    GET course metadata
+
+    Response: {
+        courseMetadata
+        oraMetadata
+        submissions
+        isEnabled
+    }
+
+    Errors:
+    - MissingParamResponse (HTTP 400) for missing params
+    - BadOraLocationResponse (HTTP 400) for bad ORA location
+    - XBlockInternalError (HTTP 500) for an issue with ORA
+    - UnknownError (HTTP 500) for other errors
+    """
+
+    @require_params([PARAM_ORA_LOCATION, PARAM_SUBMISSION_ID, PARAM_ASSESSMENT_TYPE])
+    def get(self, request, ora_location, submission_uuid, assessment_type=None, *args, **kwargs):
+
+        try:
+            init_data = {}
+
+            # Get list of submissions for this ORA
+            init_data["assessments"] = get_assessments_grades(request, ora_location, submission_uuid, assessment_type)
+            #init_data["assessments"] = TEST
+
+            response_data = FeedbackSerializer(init_data).data
```

Comment:
I need more enlightenment here. Why is this called FeedbackSerializer instead of Assessment...something? Can you explain a bit? Thanks!

Reply:
@johnvente Hello, Jhon! what do you think about change the endpoint name to Assessment...something?